packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -357,17 +361,33 @@ export class ChatWidget extends Component {
 
       let scrollButton;
       if (props.widget.showScrollToBottomButton) {
-        const label = widget.hasNewMessage ? `New Messages` : null;
+        const newMessages = {
```

Comment:
We could definitely use that rule, but I don't see anything about restricting where they are defined otherwiwse